src/utils/builder/edit_member.rs

```diff
@@ -2,27 +2,34 @@ use serde_json::builder::ObjectBuilder;
 use std::default::Default;
 use ::model::{ChannelId, RoleId};
 
+/// A builder that edits properties of a [`Member`] via
+/// [`Context::edit_member`]: ../../client/struct.Context.html
+
 pub struct EditMember(pub ObjectBuilder);
```

Comment:
Still have a blank line between the docs and the struct. And add an empty /// line between the sentence and the link